### PR TITLE
Update ticket page

### LINKED ISSUE #387: implement mockup for ticket page
The ticket page should be updated to look like the mockups below

Screenshot 2021-04-27 at 12 05 30
Screenshot 2021-04-27 at 12 05 49
Screenshot 2021-04-27 at 12 06 04

### PR DESCRIPTION
**Pull request by song50119**
Resolves #387

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -8,8 +8,6 @@ decred.org/cspp v0.3.0/go.mod h1:UygjYilC94dER3BEU65Zzyoqy9ngJfWCD2rdJqvUs2A=
 decred.org/dcrwallet v1.6.0-rc4 h1:5IT6mFa+2YMqenu6aE2LetD0N8QSUVFyAFl205PvIIE=
 decred.org/dcrwallet v1.6.0-rc4/go.mod h1:lsrNbuKxkPGeHXPufxNTckwQopCEDz0r3t0a8JCKAmU=
 dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7gZCb22OMCxBHrMx7a5I7Hp++hsVxbQ4BYO7hU=
-gioui.org v0.0.0-20210228180843-e1248651c871 h1:LDHDc7aDPdzf9I5MfEnSRCSJj6W0Nf6Dl9cLOGj4/ew=
-gioui.org v0.0.0-20210228180843-e1248651c871/go.mod h1:Y+uS7hHMvku1Q+ooaoq6fYD5B2LGoT8JtFgvmYmRzTw=
```
Comment (oshorefueled) [inline go.sum]: There's no go.mod file committed on this PR, a go.sum shouldn't be committed either.

File: ui/compontents.go [hunk 2]
```diff
@@ -473,6 +473,11 @@ func (page *pageCommon) accountSelectorLayout(gtx layout.Context, callingPage, s
 		walName := page.info.Wallets[page.wallAcctSelector.selectedReceiveWallet].Name
 		bal := page.info.Wallets[page.wallAcctSelector.selectedReceiveWallet].Accounts[page.wallAcctSelector.selectedReceiveAccount].TotalBalance
 		return d(gtx, acctName, walName, bal, page.wallAcctSelector.receivingAccountBtn)
+	case callingPage == "purchase":
+		wallSelect := page.info.Wallets[page.wallAcctSelector.selectedPurchaseTicketWallet]
+		acctName := wallSelect.Accounts[page.wallAcctSelector.selectedPurchaseTicketAccount].Name
```
Comment (oshorefueled) [inline ui/compontents.go]: `wallSelect.Accounts[page.wallAcctSelector.selectedPurchaseTicketAccount]` can be assigned to a variable ass "acc". You can then access the name as `acc.Name` and balance as  `acc.TotalBalance`

File: ui/page.go [hunk 3]
```diff
@@ -27,16 +27,43 @@ type pageIcons struct {
...
+	ticketVotedIcon2,
+	ticketVotedIcon3,
+	ticketMissedIcon1,
+	ticketMissedIcon2,
+	ticketMissedIcon3,
+	ticketExpiredIcon1,
+	ticketExpiredIcon2,
+	ticketExpiredIcon3,
+	ticketRevokedIcon1,
+	ticketRevokedIcon2,
```
Comment (oshorefueled) [inline ui/page.go]: Why do we have icons repeated with integer increments in their names? They seem to be the same icons.

File: wallet/commands.go [hunk 4]
```diff
@@ -934,41 +940,85 @@ func (wal *Wallet) SetupAccountMixer(walletID int, walletPassphrase string, errC
...
+		}
+
+		vspd := wal.multi.NewVSPD(host, walletID, accountID)
+		_, err := vspd.GetInfo()
+		if err != nil {
+			go func() {
+				errChan <- err
+			}()
+			return
+		}
```
Comment (oshorefueled) [inline wallet/commands.go]: This looks like a new vspd instance is created every time the PurchaseTicket method is called.

**Review by oshorefueled — CHANGES_REQUESTED** [review]

**Comment by oshorefueled** [comment]
Recent activity doesn't look like it's taking account of every wallet created. I have tickets that are more recent than 109 days.

Screenshot 2021-05-05 at 00 06 58

File: ui/page.go (hunk 3 above)
Comment (song50119) [reply ui/page.go]: Same icon but different size

**Review by oshorefueled — APPROVED** [review]

**Pull request merged**